source/elements/oneMKL/source/domains/blas/rot.rst

```diff
@@ -43,6 +55,10 @@ the sum of two of these scalar-vector products as follow:
         -  ``float`` 
       * -  ``double`` 
         -  ``double`` 
+      * -  ``std::complex<float>`` 
```

Comment:
I see difference with DPC++ documentation, where we support seperate T_scalarC and T_scalarS.

Reply:
Oops, thanks for the catch, fixed.